milli/src/vector/mod.rs

```diff
@@ -352,7 +352,7 @@ where
                 None,
                 deserr::ErrorKind::Unexpected {
                     msg: format!(
-                        "the distribution sigma must be in the range ]0, 1], got {}",
+                        "the distribution sigma must be in the range [0, 1], got {}",
```

Comment:
Why did you changed this?

```suggestion
                        "the distribution sigma must be in the range ]0, 1], got {}",
```

Reply:
Oh, didn't quite understand what's going on there. But now I think I do, reverting it.